### PR TITLE
Robert added robert.html

### PR DESCRIPTION
**Pull request by robsdagreat**
this is an HTML document I added

### CODE REVIEW

File: robert.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```
Comment (k3lly003) [inline robert.html:5]: @robsdagreat 
On this line change http to https

Comment (ClaironNiwemfura) [inline robert.html:5]: @robsdagreat  remove it

File: robert.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (MUNYESHURImanzi) [inline robert.html:2]: remove lang="en"

File: robert.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (renepromesse) [inline robert.html:6]: this is a normal feedback

**Review by ClaironNiwemfura — CHANGES_REQUESTED** [review]
you did a good job but check my comment

**Review by MUNYESHURImanzi — COMMENTED** [review]
good job but look after comm

**Review by k3lly003 — APPROVED** [review]
Good work!

**Review by renepromesse — COMMENTED** [review]
this is a good but please look for the comments

**Pull request merged**